iocage/lib/Jail.py

```diff
@@ -351,7 +351,7 @@ def _run_hook(self, hook_name: str):
             f"Running {hook_name} hook for {self.humanreadable_name}"
         )
 
-        lex = shlex.shlex(value)
+        lex = shlex.shlex(value)  # noqa: T484
```

Comment:
oh?

Reply:
This started spitting in all our branches. We can remove later, but it was blocking my work.

My plan is to revisit all `# noqa` occasions before we call out an alpha release.

Reply:
😎